src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -140,7 +140,7 @@ def create(
             trsvcid: int = 4420,
             adrfam: int = 0,  # IPv4
         ):
-            return NVMeoFClient().stub.create_listener(
+            return NVMeoFClient(traddr=traddr).stub.create_listener(
```

Comment:
Not sure how fast gRPC connection establishment is (plus gRPC Python class initialization), but we should probably use connection/client pooling. This can be easily implemented with the `@ttl_cache` decorator (from dashboard `plugin` dir) or any other decorator/function memoizing cache, with cache invalidation to refresh connections/cache on failure (@pereman2 implemented that for the RBD cache).

Reply:
I would like to do that in a follow up PR as it would disturb scope of this PR, thanks

Reply:
Tracker: https://tracker.ceph.com/issues/68273